nncf/onnx/quantization/quantizer_parameters.py

```diff
@@ -51,89 +47,39 @@ def get_level_low_level_high(tensor_type: np.dtype, num_bits: int) -> Tuple[int,
     return - (2 ** num_bits) // 2, (2 ** num_bits) // 2 - 1
 
 
-def calculate_scale_zero_point(max_val: np.ndarray, min_val: np.ndarray, level_low: int, level_high: int,
-                               mode: QuantizationMode, tensor_type: np.dtype) -> Tuple[np.ndarray, np.ndarray]:
+def calculate_scale_zero_point(input_low: np.ndarray, input_high: np.ndarray, level_low: int, level_high: int,
+                               mode: QuantizationMode, tensor_type: np.dtype,
+                               eps: float = 1e-6) -> Tuple[np.ndarray, np.ndarray]:
     """
     Calculates Quantizer/Dequantizer layer scale level.
-    Returns scale and zero_point values foe the quantizer.
+    Returns scale and zero_point values for the quantizer.
 
-    :param max_val: The maximum value of the input tensor.
-    :param min_val: The minimum value of the input tensor.
+    :param input_low: Minimum limit for input value.
+    :param input_high: Maximum limit for input value.
```

Comment:
IMHO, It would be more clear if we can clearly describe the difference between the `input_` and `level_` options.

Reply:
done